cli/command_restore.go

```diff
@@ -19,10 +23,18 @@ import (
 )
 
 const (
-	restoreCommandHelp = `Restore a directory or file from a snapshot into the specified target path.
+	restoreCommandHelp = `Restore from placeholder or snapshot.
```

Comment:
nit: "Restore a directory or a file" no need to bring placeholders in the first line.

Reply:
Done.